style.css

```diff
@@ -137,6 +137,11 @@ tr:hover a{
 		top:40px;
 	}
 	.leaderboard table{
-		top:60px;
+		top:10px;
 	}
-}
\ No newline at end of file
+	#repoName{
```

Comment:
Use class names when adding styles, avoid using IDs

Reply:
ohh okk so i have to change this or is it fine??